client/signup/steps/theme-selection/index.jsx

```diff
@@ -126,9 +123,9 @@ const ThemeSelectionStep = React.createClass( {
 				</div>
 				<div className={ themesWrapperClassName } >
 					<StepWrapper
-						fallbackHeaderText={ this.translate( 'Choose a theme.' ) }
-						fallbackSubHeaderText={ this.translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
-						subHeaderText={ this.translate( 'Choose a theme. You can always switch to a different theme later.' ) }
+						fallbackHeaderText={ translate( 'Choose a theme.' ) }
+						fallbackSubHeaderText={ translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
```

Comment:
That's weird, while I was checking it in PhpStorm it showed it as 151 characters long. It wasn't a lint error though, just a warning:
`126:1  warning  Line 126 exceeds the maximum line length of 140  max-len
`

Anyway, looks good to go now. Thanks for fixing it! :+1: